apps/system/js/browser_mixin.js

```diff
@@ -173,16 +173,22 @@
       }
     },
 
```

Comment:
Make a comment here
// For testing purpose; we cannot mock document.activeElement in unit test...